ops/lib.rs

```diff
@@ -0,0 +1,127 @@
+use proc_macro::TokenStream;
+use quote::quote;
+
+#[proc_macro_attribute]
+pub fn op(_attr: TokenStream, item: TokenStream) -> TokenStream {
+  let func = syn::parse::<syn::ItemFn>(item).expect("expected a function");
+  let name = &func.sig.ident;
+  let generics = &func.sig.generics;
+  let type_params = &func.sig.generics.params;
+  let where_clause = &func.sig.generics.where_clause;
+
+  let inputs = &func.sig.inputs;
+  let output = &func.sig.output;
+  let a = match &inputs[1] {
+    syn::FnArg::Typed(pat) => match *pat.pat {
+      syn::Pat::Wild(_) => quote! {
+        let a = ();
+      },
+      _ => quote! {
+        let a = args.get(2);
+        let a = deno_core::serde_v8::from_v8(scope, a).unwrap();
+      },
+    },
+    _ => unreachable!(),
+  };
+
+  let b = match &inputs[2] {
+    syn::FnArg::Typed(pat) => match *pat.pat {
+      syn::Pat::Wild(_) => quote! {
+        let b = ();
+      },
+      _ => quote! {
+        let b = args.get(3);
+        let b = deno_core::serde_v8::from_v8(scope, b).unwrap();
+      },
+    },
+    _ => unreachable!(),
+  };
+
+  // TODO(@littledivy): Optimize Result<(), Err> to skip serde_v8.
```

Comment:
I presume this should not be hard at all?

Reply:
yup, just a lot of nasty `match`-es :-)

Reply:
done, it was definitely worth it.